build/helper/metadata_add_all.py

```diff
@@ -291,13 +291,34 @@ def _add_python_name(a, attributes):
     attributes[a]['python_name'] = n
 
 
+def _add_converter_defaults(a, attributes):
```

Comment:
Why is this still needed? I would think the attribute class is enough.

Reply:
It's not. Removed